configure

```diff
@@ -152,6 +152,8 @@ Optional options:
   --disable-utmpx                   Disable manipulating the utmp/utmpx database via the related POSIX functions
   --enable-initgroups               Enable initialization of supplementary groups for run-as [Enabled]
   --disable-initgroups              Disable initialization of supplementary groups for run-as
+  --enable-selinux                  Enable SELinux support [Only avilable on Linux based systems with SELinux support]
```

Comment:
As was mentioned:
```
Defaults for the options are specified in brackets.
```
Now it is:
```
[Only avilable on Linux based systems with SELinux support]
```

That is not specifying the default, it's just a statement of availability.

Also "avilable" has a typo.